specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2021-07-01-preview/loadtestservice.json

```diff
@@ -1,2641 +0,0 @@
-{
-  "swagger": "2.0",
-  "info": {
-    "title": "CloudNativeTest",
-    "version": "2021-07-01-preview"
```

Comment:
why api version 2021-07-01-preview and 2022-06-01-preview get removed?

Reply:
They're deprecated @qiaozha .I've also shared screenshots of public communication above.